book/03-git-branching/sections/nutshell.asc

```diff
@@ -124,6 +124,16 @@ Let's switch back to the `master` branch:
 $ git checkout master
 ----
 
+[NOTE]
+.`git log` doesn't show _all_ the branches _all_ the time
+====
+If you were to run `git log` right now, you might wonder where the "testing" branch you just created went, as it would not appear in the output. 
+
+The branch hasn't disappeared; git just doesn't know that you're interested in that branch and it is trying to show you what it thinks you're interested in. In other words, by default, `git log` will only show commits below the branch you've checked out.
```

Comment:
Would you mind splitting this up so there's only one sentence per source line? It makes future diffs _much_ easier to read.

Reply:
How's that? Did I understand your request?

Reply:
Yes, that's perfect!